engine/src/main/java/org/terasology/logic/actions/TunnelAction.java

```diff
@@ -78,13 +84,13 @@ public void onActivate(ActivateEvent event, EntityRef entity) {
 
         int particleEffects = 0;
         int blockCounter = MAX_DESTROYED_BLOCKS;
-        for (int s = 0; s <= 512; s++) {
+        for (int s = 0; s <= MAX_TUNNEL_DEPTH; s++) {
```

Comment:
Fun coincidence. At first I found 512 was a good number for the level of destruction, which matched the old number for "depth" - which turned out not to matter one bit as it never made it anywhere near that deep due to the max number of blocks destroyed. Unless you turned that up to 10k or more anyway, which would make crazy deep (and laggy) tunnels.

64 there seemed a decent number, which just so happened to be the old level of destruction :-)